src/components/button/_button.scss

```diff
@@ -2,24 +2,30 @@
 
 @include exports("button") {
   .govuk-c-button {
+    // Box sizing does not work on the input
```

Comment:
I'm not sure what this comment is trying to tell me - what are the implications? It feels like it's missing a 'so we have to… [do something else as well]' clause.

Reply:
It's coming from the initial commit, I should remove it now, as it's not longer the case.

Reply:
👍